player/public/services/interfaceService.js

```diff
@@ -235,6 +235,10 @@ org.ekstep.service.content = new (org.ekstep.service.mainService.extend({
 	setUser: function (uid) {
 		return new Promise(function (resolve, reject) {
 			org.ekstep.service.renderer.setUser(uid).then(function (data) {
+				// It will reload the app for new user once he gets out of player page.
+				if(window.cordova) {
+					window['playerActionHandlerDelegate'].onUserSwitch({uid: uid});
```

Comment:
window['playerActionHandlerDelegate'] 
What is this method? I didn't see this method previously. Why this new logic now & who agreed to change this?

Reply:
This is the callback for the mobile when the player changes the user So that when user will close the player mobile will reload the app for that new user.